### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by k4nar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+For example, let's say you have the following model:
+```python
+from django.db import models
+
+from project.ltree import LtreeField
+
+
+class Category(models.Model):
+  parent = models.ForeignKey('self', null=True)
+  code = models.CharField(maxlength=32, unique=True)
```
Comment (ewjoachim) [inline README.md:44]: wow, is this limiting the size of the tree on the amount of levels you can fit in a 32 chars code ? :D A rather bold choice :D

File: README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+[`demo/categories/sql/triggers.sql`](/demo/categories/sql/triggers.sql).
+
+
+## The demo
+
+In the demo, the following files are the most important:
+- [`demo/categories/models.py`](/demo/categories/models.py): The definition of
+  a model using ltree
+- [`demo/categories/ltree.py`](/demo/categories/ltree.py): A very simple Django
+  field for ltree. More lookups could be added (one for `~` for instance).
```
Comment (ewjoachim) [inline README.md:89]: Did you know : the `@>` operator has already been implemented in django as `__contains` as of Django >= 1.8 (https://github.com/django/django/blob/stable/1.8.x/django/contrib/postgres/lookups.py). and `<@` as `contained_by`, so you could do the same without defining custom lookups :)

**Review by ewjoachim — APPROVED** [review]
👏  I ❤️ it !

File: README.md (hunk 1 above)
Comment (k4nar) [reply README.md:44]: Nope, here code is the unique identifier of each node. There is no limit on the total as `path` is a text field.

File: README.md (hunk 2 above)
Comment (k4nar) [reply README.md:89]: Yes, but I find `ancestor` and `descendant` more convenient though.

Comment (ewjoachim) [reply README.md:89]: (or you could inherit from the existing lookups and just redifine their lookup_name if you really want to use "ancestor")

File: README.md (hunk 1 above)
Comment (ewjoachim) [reply README.md:44]: ah ok:)

**Pull request merged**